idp/idpclient/client.go

```diff
@@ -0,0 +1,171 @@
+// name 'idpclient' is used although it repeats the name of the outer package 'idp'
```

Comment:
Do we need to explain this? Or should we instead explain in a short sentence what the idpclient package does?